Source/HYPScatterPlot.m

```diff
@@ -74,107 +80,109 @@ - (instancetype)initWithFrame:(CGRect)frame
                               @"upperThresholdYLabel": self.upperThresholdYLabel,
                               @"lowerThresholdYLabel": self.lowerThresholdYLabel,
                               @"plotView": self.plotView};
-
+    
     [self addConstraints:
      [NSLayoutConstraint constraintsWithVisualFormat:@"H:|-padding-[minimumXLabel]"
                                              options:0
                                              metrics:metrics
                                                views:views]];
-
+    
```

Comment:
@jacob-andersson haven't you recently upgraded Xcode? that might break existing plug-ins